src/main/java/ua/vedroid/cinema/service/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package ua.vedroid.cinema.service.mapper;
+
+public interface DtoMapper<E, Q, P> {
```

Comment:
Interface Segregation. I actually don't think it is a good idea to make a single generic interface. Maybe we should do two interfaces, cause in the future we'll have situations, where a model won't need either ResponseDto, or RequestDto

Reply:
In this case, you can use generics like DtoMapper<Movie, MovieDto, MovieDto>